samples/BikeSharingApp/Bikes/server.js

```diff
@@ -11,7 +11,8 @@ var async = require('async');
 
 var mongoDBDatabase = process.env.mongo_database || "admin";
 var mongoDBCollection = process.env.mongo_collection || "bikes";
-var mongoDBConnStr = process.env.mongo_connectionstring || "mongodb://databases-mongo";
+var mongoDBConnStrHost = process.env.mongo_connectionstring || "mongodb://databases-mongo";
+var mongoDBConnStr = mongoDBConnStrHost + ":" + process.env.DATABASES_MONGO_SERVICE_PORT;
```

Comment:
How is this variable set?

Reply:
This is a Kubernetes env var. On second thought, I can update this to process.env.DATABASES_MONGO_SERVICE_PORT || 27017
if users want to setup mongo locally

Reply:
Are you talking about DATABASES_MONGO_SERVICE_PORT? I think it is just a default Kubernetes service environment variable.
If you're talking about mongo_connectionstring, it's declared here: https://github.com/microsoft/mindaro/blob/ceaa670f4d684a8e2f3fdcdb4e15afaecf24faba/samples/BikeSharingApp/Bikes/charts/bikes/values.yaml

Reply:
Thanks guys for the responses. It makes sense to add a default value as well for local debugging.